funcframework/framework.go

```diff
@@ -99,33 +117,38 @@ func registerEventFunction(path string, fn interface{}, h *http.ServeMux) {
 	})
 }
 
+func registerCloudEventFunction(path string, fn func(cloudevents.Event), h *http.ServeMux) {
+	p, err := cloudevents.NewHTTP()
+	if err != nil {
+		log.Fatalf("failed to create protocol: %s", err.Error())
+	}
+
+	handleFn, err := cloudevents.NewHTTPReceiveHandler(context.Background(), p, fn)
+
+	if err != nil {
+		log.Fatalf("failed to create handler: %s", err.Error())
```

Comment:
It's not clear to me that these should be `log.Fatal`. If this is going to fail, does it fail every time? Seems like it's something that could fail for other reasons.

Reply:
What should it be, then? `log.Error` and then `return`?

Reply:
I was thinking returning the error up the stack.